lib/snowplow-emr-etl-runner/emr_job.rb

```diff
@@ -48,7 +48,7 @@ class EmrJob
       SHRED_STEP_OUTPUT = 'hdfs:///local/snowplow/shredded-events/'
 
       SHRED_JOB_WITH_PROCESSING_MANIFEST = Gem::Version.new('0.14.0-rc1')
-      SHRED_JOB_WITH_TSV_OUTPUT = Gem::Version.new('0.16.0-rc1')
```

Comment:
I'd stay with rc1 because it was first factual version with TSV. Just in case some lost soul decided to stick to RC assets.

Reply:
If 0.16.0-rc1 is the first version with TSV, it means that *all* 0.16.0 versions are with TSV, doesn't it?

Reply:
You were right. EER is smarter that I thought it was to compare versions, I thought that it was alphabetical comparison but no it's able to understand RC. On biz1 we're using -rc7 and it didn't work with `0.16.0`.